spec-dtslint/observables/of-spec.ts

```diff
@@ -0,0 +1,55 @@
+import { of, animationFrameScheduler } from 'rxjs';
+import { a, b, c, d, e, f, g, h, i, j } from '../helpers';
+
+it('should infer correctly with 1 param', () => {
+  const res = of(a); // $ExpectType Observable<Observable<A>>
+});
+
+it('should infer correcly with mixed type of 2 params', () => {
+  const res = of(a, b); // $ExpectType Observable<Observable<A> | Observable<B>>
```

Comment:
@cartant 
This is what I was talking about - To me it should be `Observable<A | B>`

Reply:
That's because `a` and `b`, etc. are already observables.

```ts
const res = of(new A(), new B()); // $ExpectType Observable<A | B>
```

Reply:
I have changed them to reflect the typings appropriately. Thanks.